src/sql/parts/modelComponents/card.css

```diff
@@ -110,6 +113,30 @@
 	border-style: solid;
 }
 
+.model-card-list-item .selection-indicator-container,
+.model-card .selection-indicator-container {
+	position: absolute;
+	overflow: hidden;
+	width: 16px;
+	height: 16px;
+	border-radius: 50%;
+	background-color: white;
```

Comment:
Does this color need to be theme compliant? I think this CSS property and border-color should probably be in something that updates on theme change. We have lots of examples doing this in the code.

Reply:
the colors used here are theme compliant, at least for the light and dark theme that we officially support.